packages/cli/src/lib/snapshot/synchronization/synchronizationPlan.ts

```diff
@@ -0,0 +1,29 @@
+import {ResourceSnapshotsSynchronizationPlanModel} from '@coveord/platform-client';
+
+export class SynchronizationPlan {
+  public constructor(
+    public readonly model: ResourceSnapshotsSynchronizationPlanModel
+  ) {}
+
+  public containsUnambiguousMatches(): boolean {
+    const synchronizationOperations =
+      this.model.resourceSynchronizationOperations;
+    if (!synchronizationOperations) {
+      return false;
+    }
+
+    for (const [_, operations] of Object.entries(synchronizationOperations)) {
```

Comment:
```suggestion
    for (const operations of Object.values(synchronizationOperations)) {
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/values

Reply:
yeah, I kept it like this because I intended to use the keys in the synchronization preview... but I can change it later